docs/conf.py

```diff
@@ -66,13 +67,21 @@ def convert_markdown_mathjax_for_rst(lines: List[str]) -> List[str]:
     return result
 
 
-def pandoc_process(app,
-                   what: str,
-                   name: str,
-                   obj: Any,
-                   options,
-                   lines: List[str]
-                   ) -> None:
+def autodoc_skip_member(
+        app,
+        what: str,
+        name: str,
+        obj: Any,
+        skip: bool,
+        options,
+) -> bool:
+    """Public members already kept. Also include members marked as documented.
+    """
+    return id(obj) not in _doc.RECORDED_CONST_DOCS
```

Comment:
This returns true for all id(objs) not in this special list, and therefor skips anything not in this special set.  This removes documentation when it shouldn't.